src/App.js

```diff
@@ -1,13 +1,79 @@
 import React from 'react';
-
 import './App.scss';
+import GoodsList from './components/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  loadAll = () => {
+    getAll()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
+        });
+      });
+  }
+
+  loadFive = () => {
+    get5First()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
+        });
+      });
+  }
+
+  loadRed = () => {
+    getRedGoods()
+      .then((goodsFromServer) => {
+        this.setState({
+          goods: goodsFromServer,
```

Comment:
You could use 1 function instead of these 3, and it could accept a callback as a parameter